VPM.py

```diff
@@ -215,11 +228,33 @@ def ffts_to_mel(ffts, win_length=1024, overlap=.5, n_mels=256,
     Check out librosa.filters.mel if unsure how to write the arguments to call 
     librosa.feature.melspectrogram.
     """
+    D = np.abs(ffts) ** 2
+    mel_freq_spec = librosa.feature.melspectrogram(S=D, sr=sample_rate, n_mels=n_mels)
+
+    if plot:
+        plt.figure(figsize=(10, 4))
+        S_dB = librosa.power_to_db(mel_freq_spec, ref=np.max)
+        librosa.display.specshow(S_dB, x_axis='time', y_axis='mel', sr=sample_rate, fmax=sample_rate/2.0)
+        plt.colorbar(format='%+2.0f dB')
+        plt.title('Mel-frequency spectrogram')
+        plt.tight_layout()
+        plt.show()
+
+
     if not skip_mfcc:
         print("Computing MFCC")
-        """
-        !! Write code to compute MFCC here !!
-        """
+        mfccs = librosa.feature.mfcc(S=librosa.power_to_db(mel_freq_spec))
+
+        if plot:
+            plt.figure(figsize=(10,4))
+            librosa.display.specshow(mfccs, x_axis='time')
+            plt.colorbar()
+            plt.title('MFCC')
+            plt.tight_layout()
+            plt.show()
+
+    # return 2 parameters?
```

Comment:
Remove the comment when done, yes, it is normal in python to return a tuple of values. 

Please put two separate return statements, so that we don't call return mfccs when `skip_mfcc` is true, otherwise we return an uninitialized variable..